src/main/java/bisq/desktop/main/offer/EditableOfferDataModel.java

```diff
@@ -245,7 +245,7 @@ public boolean initWithData(OfferPayload.Direction direction, TradeCurrency trad
         fillPaymentAccounts();
 
         PaymentAccount account;
-        PaymentAccount lastSelectedPaymentAccount = preferences.getSelectedPaymentAccountForCreateOffer();
+        PaymentAccount lastSelectedPaymentAccount = getPreselectedPaymentAccount();
```

Comment:
Can we add @Nullable annotations for all nullable fields?

Reply:
Done